src/renderer/features/wallets/WalletSelect/model/__tests__/wallet-select-model.test.ts

```diff
@@ -110,28 +110,14 @@ describe('features/wallets/WalletSelect/model/wallet-select-model', () => {
     const scope = fork();
 
     expect(scope.getState(walletModel.$activeWallet)).toEqual(undefined);
-    await allSettled(walletModel.$allWallets, { scope, params: wallets });
+    await allSettled(walletModel._test.$allWallets, { scope, params: wallets });
 
     expect(spyRead).not.toHaveBeenCalled();
     expect(spyUpdateAll).not.toHaveBeenCalled();
     expect(spyUpdate).not.toHaveBeenCalled();
     expect(scope.getState(walletModel.$activeWallet)).toEqual(wallets[0]);
   });
 
-  test('should set $activeWallet when $wallets receives new wallet', async () => {
```

Comment:
maybe update this test so we can still have this logic covered?

Reply:
a lot of this tests have a assumption, that direct change of $allWallet store should change some relative values. It's not true in the new code because some values are not derived anymore